library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -358,7 +370,12 @@ libspdm_return_t libspdm_build_response(void *context, const uint32_t *session_i
     if (session_id != NULL) {
         libspdm_get_scratch_buffer (spdm_context, (void **)&scratch_buffer, &scratch_buffer_size);
         my_response = scratch_buffer + transport_header_size;
-        my_response_size = scratch_buffer_size - transport_header_size;
+        #if LIBSPDM_ENABLE_CAPABILITY_CHUNK_CAP
+        my_response_size = scratch_buffer_size - transport_header_size - LIBSPDM_MAX_SPDM_MSG_SIZE -
+                           LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE;
+        #else
+        my_response_size = scratch_buffer_size - transport_header_size
+        #endif
```

Comment:
Please add comment into the code:
The rang of 0 ~ LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE is for Secure encoding/
decoding, LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE ~ End is for Chunk.